.config/jest.config.js

```diff
@@ -26,21 +27,40 @@ async function loadRunConfig(cfgPath) {
     return DefaultRunCfg
 }
 
-function resolveTestMatches(runCfg) {
+function resolveTestMatches(HOME_DIR, runCfg) {
     return runCfg.match.map(
         p => {
             if (path.isAbsolute(p)) {
                 return p
             }
-            return path.join(runCfg.projectPath, p)
+            return path.join(HOME_DIR, runCfg.projectPath, p)
```

Comment:
The project path is absolute. No need to join with the home directory.

Reply:
@alexplischke I fixed it ... I noticed that the `run.yaml` 's in Cypress allow relative paths for `projectPath` (see https://github.com/saucelabs/sauce-cypress-runner/blob/master/tests/config-tests/run.yaml). So I just added a check that if it isn't absolute, make it absolute.